src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,49 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0 30px 0;
+
+  &__social-media {
+    display: flex;
+    justify-content: space-between;
+    width: 120px;
+    margin-bottom: 33px;
+    transition: 0.5s;
```

Comment:
The same, always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.

Reply:
Still need to change)